typings/index.d.ts

```diff
@@ -3,7 +3,8 @@
 declare module 'draft-js-export-html' {
     import draftjs = require("draft-js");
 
-    type BlockStyleFn = (block: draftjs.ContentBlock) => any;
+    type BlockStyleFn = (block: draftjs.ContentBlock) => RenderConfig;
+    type EntityStyleFn = (entity: draftjs.EntityInstance) => RenderConfig;
```

Comment:
I'm not very familiar with TypeScript but I think this should be a nullable return type (a "maybe type").